runtime/near-vm-logic/src/logic.rs

```diff
@@ -2424,8 +2436,20 @@ impl<'a> VMLogic<'a> {
         let nodes_before = self.ext.get_trie_nodes_count();
         let read = self.ext.storage_get(&key);
         let nodes_delta = self.ext.get_trie_nodes_count() - nodes_before;
+        #[cfg(feature = "io_trace")]
+        let (tn_mem_reads, tn_db_reads) = (nodes_delta.mem_reads, nodes_delta.db_reads);
```

Comment:
Style nit: Inline these variables to avoid having two distinct `cfg`s?

Reply:
`nodes_delta` was moved before the event is emitted. I changed it to a borrow now, which makes it possible to simplify as you suggested.